addons/hr_expense/models/hr_expense_sheet.py

```diff
@@ -750,7 +750,8 @@ def _get_expense_account_destination(self):
         if self.payment_mode == 'company_account':
             journal = self.payment_method_line_id.journal_id
             account_dest = (
-                journal.outbound_payment_method_line_ids[:1].payment_account_id
+                self.payment_method_line_id.payment_account_id
+                or journal.outbound_payment_method_line_ids[:1].payment_account_id
```

Comment:
Couldn't we remove `journal.outbound_payment_method_line_ids[:1].payment_account_id`?
if we already have a `payment_method_line_id` it doesn't make any sense to take the first `outbound_payment_method_line_ids` from it's journal

Reply:
I agree, it's not really meaningful